packages/rich-text/src/test/remove-previous-line-separator.js

```diff
@@ -0,0 +1,88 @@
+/**
+ * External dependencies
+ */
+import deepFreeze from 'deep-freeze';
+
+/**
+ * Internal dependencies
+ */
+
```

Comment:
There's no consistency to the newline placement after these blocks. While technically against the standard, most else of the application omits the newline. We should at the very least be consistent within the same file.

Reply:
Yes. I thought there should be a newline in between, so I try to be consistent with that. It would be good to have a linting rule for it.